src/elements/content-sidebar/activity-feed/annotation-thread/messages.js

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    errorFetchAnnotation: {
+        id: 'be.annotattionThread.errorFetchAnnotation',
+        description: 'Error message when an annotation fetch fails',
+        defaultMessage: 'Cannot fetch annotation.',
```

Comment:
Minor, but our grammar is a bit inconsistent between errors. Perhaps `The annotation could not be fetched`?